kong/clustering/rpc/json_rpc_v2.lua

```diff
@@ -23,7 +23,10 @@ local ERROR_MSG = {
 
 
 function _M.new_error(id, code, msg)
-  if not msg then
+  if msg then
+    msg = assert(tostring(msg))
```

Comment:
We might expose something like `table: 0x7fff0000` to the caller.

Reply:
Could you give the suggest code?

Reply:
https://github.com/Kong/kong/pull/12951#discussion_r1582009060

Reply:
> Could you give the suggest code?

https://github.com/Kong/kong/pull/12951#discussion_r1582009060